docs/rbac.yaml

```diff
@@ -0,0 +1,63 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: kube-aws-iam-controller
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kube-aws-iam-controller
+rules:
+- apiGroups:
+  - "amazonaws.com"
+  resources:
+  - awsiamroles
+  - awsiamroles/status
+  verbs:
+  - get
+  - list
+  - watch
+  - update
+  - patch
+- apiGroups:
+  - ""
+  resources:
+  - pods
```

Comment:
Why does the controller watch pods? Is this to help clean up orphaned (no longer referenced) _Secrets_? Is it to decide whether to keep renewing credentials for _Secrets_ that are no longer referenced?

Reply:
The old support for generating secrets based on the `aws-iam-` prefixed secret mount is still supported here. I would remove it in a later PR to not break existing users without a possibility of an upgrade path.